plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -312,6 +312,8 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
           "Size of a single rocksdb memtable (in MiB)")
          ("persistent-storage-bytes-per-sync", bpo::value<uint64_t>()->default_value(config::default_persistent_storage_bytes_per_sync),
           "Rocksdb write rate of flushes and compactions.")
+         ("persistent-storage-mbytes-snapshot-batch", bpo::value<uint32_t>()->default_value(config::default_persistent_storage_mbytes_batch),
+          "Rocksdb batch size threshold before writing read in snapshot data to database.")
 
```

Comment:
Do we really need a knob for something only used when reading a snapshot?

Reply:
The intent was that having this means that if someone is using rocksdb because they have limited resources, that they won't be forced to have as high of memory available the we default to, but that if someone has plenty of memory, that they can jack it way up and start faster.
@tbfleming @larryk85 @b1bart

Reply:
I don't see any harm in having the knob.  And it would allow for operators with more robust machines to set the value higher and read the snapshots in as fast as possible (bringing a reserve node up to parity or a main node back after failure).

Reply:
How much faster is it when turning the knob up?

Reply:
I prefer to have a knob; fill-rocksdb had perf gains with high batch sizes (somewhere between 100-150 blocks of full-history deltas on mainnet). I no longer have the numbers, but the difference was large.

Reply:
I'll defer. I just believe we need to question every new knob as each and every one means additional testing and potential additional error reports when used incorrectly.

Reply:
My measurements with a mainnet snapshot:
50MB: 63s
200MB: 63s
1024MB: 61s